packages/app/src/app/components/Preview/index.js

```diff
@@ -217,15 +220,12 @@ class BasePreview extends React.Component<Props, State> {
         frameInitialized: false,
       });
       if (this.$socket) {
+        this.state.localClose = true;
```

Comment:
I'm not sure if this is a good way of altering state. If there's a direct assignment I think it makes sense to not put it on state but as an instance field on the class.